app/script/conversation/ConversationRepository.js

```diff
@@ -547,11 +552,7 @@ z.conversation.ConversationRepository = class ConversationRepository {
       })
       .catch(error => {
         if (error.type !== z.conversation.ConversationError.TYPE.NOT_FOUND) {
-          this.logger.log(
-            this.logger.levels.ERROR,
-            `Failed to get conversation '${conversation_id}': ${error.message}`,
-            error
-          );
+          this.logger.error(`Failed to get conversation '${conversation_id}': ${error.message}`, error);
```

Comment:
`this.logger.levels.ERROR` is not needed anymore?

Reply:
No. This `this.logger.log(this.logger.levels,ERROR)` is the same as `this.logger.error()`.

Reply:
Same for `info`, `warning`?

Reply:
Si senor